OpenRA.Mods.Cnc/Traits/SupportPowers/DropPodsPower.cs

```diff
@@ -0,0 +1,158 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Activities;
+using OpenRA.GameRules;
+using OpenRA.Mods.Cnc.Effects;
+using OpenRA.Mods.Common;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Orders;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Cnc.Traits
+{
+	public class DropPodsPowerInfo : SupportPowerInfo, IRulesetLoaded
+	{
+		[Desc("DropPod Unit")]
+		[ActorReference(typeof(AircraftInfo))]
+		public readonly string UnitType = "badr";
```

Comment:
Speaking from experience, setting the internal default to null and forcing modders to set the unit type explicitly via `[FieldLoader.Require]` causes the least problems, especially in case this ever needs an update rule.